src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/FindProblemMessageJobTest.java

```diff
@@ -77,10 +82,28 @@ void should_handle_ccd_case_event_asb_message_when_return_old_ready_messages() {
         assertEquals(MESSAGE_ID, readyMessages.get(0));
     }
 
+    @ExtendWith(OutputCaptureExtension.class)
+    @Test
+    void should_include_case_type_id_in_log(CapturedOutput log) {
+        final CaseEventMessageEntity mockCaseEventMessageEntity = createMockCaseEventMessageEntity(MessageState.READY);
+        when(caseEventMessageRepository.findProblemMessages(messageTimeLimit))
+            .thenReturn(Collections.singletonList(mockCaseEventMessageEntity));
+        List<String> caseEventMessages = findProblemMessageJob.run();
+        assertTrue(log.getOut().contains("\"caseTypeId\" : \"CaseType_123\""));
```

Comment:
running the entire test suite from the terminal fails on the assertTrue. Whist this may pass if you execute the test singularly in intellij, this is inconsistent as suite. Can we change this to assertThat?